decidim-core/app/assets/stylesheets/decidim/modules/_comments.scss

```diff
@@ -29,20 +26,6 @@ $comment-form-bg: $light-gray;
   color: $muted;
 }
 
-.show-more--comment-thread{
```

Comment:
Even though these are unused *right now*, they are used in the design mockups and might be implemented in the future.

Reply:
Aha, let me bring them back then!

Reply:
However, I don't think we should be serving unused CSS, just like I don't believe in "commented out code just in case I uncomment it back", but with the added downside that these are bytes actually served over the network. I'd say we should _at least_ comment them out so that they're not included in the final CSS?